addons/account/models/account_move_line.py

```diff
@@ -2543,7 +2543,6 @@ def action_open_business_doc_from_aml(self):
             'name': name,
             'type': 'ir.actions.act_window',
             'view_mode': 'form',
-            'view_id': self.env.ref('account.view_move_form').id,
```

Comment:
> In the past, the view given in the context was problematic, so now we let the orm pick the view

What was the problem?

Reply:
If i remember correctly, the flow was that when we are in the bank reconciliation widget, clicking on view on a customer payment, we are redirected to the payment and when we click on the reconcile items button we had a traceback 
"AttributeError: type object 'account.payment' has no attribute 'open_reconcile_view'" 
Letting the orm pick the view by himself fixed it (advice given by Laurent)

Reply:
Ok, makes sense.

But this function [1] makes the function `open_move` and `open_payment_view` useless.
Let's remove useless code while there is still time and it is still a beta freeze :)


[1] `action_open_business_doc_from_aml`, badly named... of course it's from a aml, we are declaring it on the damn model... No need to tell it's entire life